app/src/main/java/org/sopt/official/feature/auth/AuthActivity.kt

```diff
@@ -74,106 +106,226 @@ class AuthActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        if (dataStore.accessToken.isNotEmpty()) {
-            startActivity(
-                HomeActivity.getIntent(this, HomeActivity.StartArgs(UserStatus.of(dataStore.userStatus)))
-            )
-        }
-        setContentView(binding.root)
-        initNotificationChannel()
+        setContent {
+            SoptTheme {
+                val context = LocalContext.current
+                val lifecycleOwner = LocalLifecycleOwner.current
 
-        initUi()
-        initAnimation()
-        collectUiEvent()
-    }
+                val action by viewModel.loginDialogAction.collectAsStateWithLifecycle()
```

Comment:
1.
```suggestion
                val loginDialogAction by viewModel.loginDialogAction.collectAsStateWithLifecycle()
```

2. 저게 액션일까요?
3. collectAsStateWithLifecycle이 필요할까?
4. ViewModel에 변수로 선언해야하는가?